api/proto/teleport/workloadidentity/v1/issuance_service.proto

```diff
@@ -0,0 +1,151 @@
+// Copyright 2024 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.workloadidentity.v1;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "teleport/workloadidentity/v1/attrs.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/workloadidentity/v1;workloadidentityv1pb";
```

Comment:
We've been following the buf managed mode convention for new proto packages, which should make this go package name `workloadidentityv1`, not `workloadidentityv1pb`.

```suggestion
option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/workloadidentity/v1;workloadidentityv1";
```

Reply:
[57cb7aa](https://github.com/gravitational/teleport/pull/50163/commits/57cb7aaed8e5339ea968c544cef7e3853b129c60)